l10n_it_codici_carica/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2017 Alessandro Camilli - Openforce
+#
+#    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```

Comment:
Replace © by Copyright and align License to other words.
See https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L1-L3 as reference.